violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/behavior/EditSelectedDuplicateBehavior.java

```diff
@@ -66,7 +66,7 @@ private String getNodeName(INamedNode node)
     }
 
     private final String NAME_OF_DETECTED_PROPERTY = "name";
-    private final String STEREOTYPE_ENDING = "�";
+    private final String STEREOTYPE_ENDING = "»";
```

Comment:
Write this character in the unicode format e.g \uXXXX
https://unicode-table.com/en/